examples/lujain_work_area/File_Directory/organize_file.py

```diff
@@ -0,0 +1,38 @@
+import os
+import shutil
+
+DIRECTORY = "./File_Directory"
+
+def organize_files(directory):
```

Comment:
This method is creating dirs and then organizing files. Handling 2 responsibilities. 
Later if we want to have a method that can just create multiple dirs, we would have to separate it and then sue it somewhere else.
Can we make it handle only one responsibility and create another func to handle the other responsibility.